client/src/components/Vote.svelte

```diff
@@ -36,36 +37,46 @@
   <p class="countdown mdc-typography--headline2 {countdown < 10 ? 'error' : ''}">
     {formatTimer(Math.max(countdown, 0))}
   </p>
+  <p class="mdc-typography--headline4">Vote out the catfish</p>
   <div class="voting-grid">
-    {#each lobby.players as { avatar, displayName, votes }, i}
+    {#each lobby.players as { avatar, displayName, votes, alive, promptAnswer }, i}
       <div class="vote-container">
         <button
-          class="avatar"
-          disabled={$user?.uid == lobby.uids[i]}
-          on:click={() => addVote(lobbyCode, $user?.uid ?? "", lobby.uids[i])}
+          class="avatar {!alive ? 'dead' : ''} {selected == i ? 'selected' : ''}"
+          disabled={!alive}
+          on:click={async () => {
+            await addVote(lobbyCode, $user?.uid ?? "", lobby.uids[i]);
+            selected = i;
+          }}
         >
-          <img src="/avatars/{avatar}.webp" alt={avatarAltText[avatar]} />
+          <img class={!alive ? "dead" : ""} src="/avatars/{avatar}.webp" alt={avatarAltText[avatar]} />
           <span class="mdc-typography--subtitle1">{displayName ?? ""}</span>
+          <div class="mdc-typography--caption">
+            {#if alive}
+              {promptAnswer ?? "no answer"}
```

Comment:
For living players prompt answer here, maybe we want to indicate it somehow in addition to the smaller text. I'm thinking putting something like "Answer: their_answer" or maybe changing the color of the text to something other than the default